content/guides/installation.md

```diff
@@ -32,24 +32,24 @@ O processo de instalação solicita as seguintes escolhas.
 #### Estrutura do projeto
 Você pode escolher uma entre as seguintes estruturas de projeto:
 
-- `web` estrutura de projeto ideal para criar as clássicas aplicações renderizadas no servidor. Configuramos o suporte à sessões e instalamos o AdonisJS template engine.
-- `api` estrutura de projeto ideal para criar um servidor de API.
-- `slim` estrutura de projeto que cria a menor aplicação AdonisJS possível e não instala nenhum pacote adicional, exceto o núcleo do framework.
+- A estrutura de projeto `web` é ideal para criar as clássicas aplicações renderizadas no servidor. Configuramos o suporte à sessões e instalamos o AdonisJS template engine.
+- A estrutura de projeto `api` é ideal para criar um servidor de API.
+- A estrutura de projeto `slim` cria a menor aplicação AdonisJS possível e não instala nenhum pacote adicional, exceto o núcleo do framework.
```

Comment:
Se o objetivo for manter as palavras chaves no início da frase, podemos reescrever para:
```
api é a estrutura de projeto ideal para criar um servidor de API.
```

Reply:
Acho essa forma que sugeriu melhor:

api é a estrutura de projeto ideal para criar um servidor de API.